common/model/organization.js

```diff
@@ -17,6 +18,7 @@ export type Organization = {|
   twitterHandle: string,
   sameAs: Array<string>,
   openingHoursSpecification: OpeningHoursDay[],
+  specialOpeningHoursSpecification?: ?SpecialOpeningHours[],
```

Comment:
See the note on making array types not maybes, but rather empty arrays.
https://github.com/wellcometrust/wellcomecollection.org/pull/2402/files#r177443111